conjecture-rust/src/engine.rs

```diff
@@ -36,6 +41,7 @@ struct MainGenerationLoop {
     sender: SyncSender<LoopCommand>,
     max_examples: u64,
     random: ChaChaRng,
+    skip_phases: Vec<Phase>,
```

Comment:
At the moment, `Phase` is zero-sized and this vector won't heap allocate, but in the future, it might make sense to use a bitmap here.